Packages/io.chainsafe.web3-unity.web3auth/Runtime/chainsafe.web3-unity.web3auth.asmdef.meta

```diff
@@ -1,11 +1,6 @@
 fileFormatVersion: 2
-<<<<<<<< HEAD:src/UnitySampleProject/Assets/Scenes/SampleMain/Category.prefab.meta
 guid: 065b2a8044c813947b0e4d52126f5a26
-PrefabImporter:
-========
-guid: d0e5603e6af957942922f5d22bbbfa0c
```

Comment:
what is this removed assembly?

Reply:
unity must've done it. lemme see

Reply:
![image](https://github.com/user-attachments/assets/d312fb4f-838f-4322-927c-c21225424060)
everything seems to be alrightt there